Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -521,6 +523,10 @@ DECLARE_SOA_EXTENDED_TABLE(TracksExtra_000, StoredTracksExtra_000, "TRACKEXTRA",
 DECLARE_SOA_EXTENDED_TABLE(TracksExtra_001, StoredTracksExtra_001, "TRACKEXTRA", //! Additional track information (clusters, PID, etc.)
                            track::v001::DetectorMap);
 
+DECLARE_SOA_TABLE_FULL(TracksExtraRun2, "TracksExtraRun2", "AOD", "TRACKEXTRARUN2",
```

Comment:
I think we can simply use DECLARE_SOA_TABLE here. 
Also at the moment the name is not canonical which means that merging indices will not work. The table has no indices at present, so this is not an issue but it is anyway not ideal.
To have this canonical you need to call it TrackExtraRun2s so it matches TRACKEXTRARUN2 (which does not have the s)